builtin/settingtypes.txt

```diff
@@ -112,6 +112,10 @@ invert_mouse (Invert mouse) bool false
 #    Mouse sensitivity multiplier.
 mouse_sensitivity (Mouse sensitivity) float 0.2 0.001 10.0
 
+#    ADS mouse sensitivity multiplier.
```

Comment:
It's not a multiplier but an exponent.
Might you also be so nice to add a description? It is unclear what this abbreviation means and what the setting is good for (or used for).